R/zzz-class-model-components.R

```diff
@@ -152,12 +155,6 @@ nido <- function() {
   NIDO(model = "nido")
 }
 
-#' @rdname measurement-model
-#' @export
-ncrum <- function(max_interaction = Inf) {
-  NCRUM(model = "ncrum", list(max_interaction = max_interaction))
-}
-
```

Comment:
We should still have an `ncrum()` function for users to specify an NC-RUM model.